### ISSUE TITLE
Where is default_cfg?

### ISSUE THREAD

**Issue by QAiStudio** [zju3dv/MatchAnything#34]
Hello
from src.loftr import LoFTR, default_cfg

# The default config uses dual-softmax.
# The outdoor and indoor models share the same config.
# You can change the default values like thr and coarse_match_type.
matcher = LoFTR(config=default_cfg)
matcher.load_state_dict(torch.load("weights/outdoor_ds.ckpt")['state_dict'])

Where is default_cfg?
What does "outdoor_ds.ckpt" refer to - matchanything_eloftr.ckpt/matchanything_roma.ckpt or something else? I'm searching on Hugging Face to find
